include/grpc/grpc_security.h

```diff
@@ -240,6 +240,11 @@ GRPCAPI grpc_channel_credentials* grpc_ssl_credentials_create(
     const char* pem_root_certs, grpc_ssl_pem_key_cert_pair* pem_key_cert_pair,
     const verify_peer_options* verify_options, void* reserved);
 
+GRPCAPI void grpc_ssl_credentials_set_min_tls_version(
```

Comment:
Please add a comment for both these APIs. Please follow the style of comments used in this file.